README.rst

```diff
@@ -40,31 +40,27 @@ Next step is to install assets (js, css etc.):
     plupload
     ...
 
-Now you can follow the standard invenio installation and development procedures, such as running `inveniomanage collect`.
+Now you can follow the standard Invenio installation and development procedures, such as running `inveniomanage collect`.
 
 ---------------------
 Demosite installation
 ---------------------
-As INSPIRE uses `Mixer <https://github.com/klen/mixer>`_ to load database fixtures, you should install the library:
```

Comment:
I might be totally wrong, but isn't it installed by INSPIRE? See: https://github.com/inspirehep/inspire-next/blob/a3a56cce87c98bb02146d85679e62aeec0d70419/setup.py#L60

In any case, running that command inside the `inspire-next` folder resulted in an obscure error, since INSPIRE's `setup.py` has no `extras_require` section.